### ISSUE TITLE
Parsing issue with Void as parameter

### ISSUE THREAD

**Issue by jandamm** [nicklockwood/SwiftFormat#321]
Hi there,

while writing test code I wrote this line:
```swift
func ifFailure(_: Void) { array.append(.ifFailure) }
```
While this compiles it gets converted to:
```swift
func ifFailureVoid { array.append(.ifFailure) }
```

For now I got rid of it by implementing my own `Void` named `Failure` which works fine.
```swift
func ifFailure(_: Failure) { array.append(.ifFailure) }
```

Thanks for SwiftFormat 👍

**Comment by nicklockwood** [comment]
@jandamm thanks for reporting - I'll look into it.

**Comment by nicklockwood** [comment]
@jandamm I've pushed a fix for the Void rule to the develop branch if you'd like to try it out.

**Comment by jandamm** [comment]
Looks good, thanks 🙂

**Comment by nicklockwood** [comment]
@jandamm fix landed in version 0.35.8
